src/components/SelectedCardList/SelectedCardList.tsx

```diff
@@ -7,12 +8,16 @@ type Props = {
 }
 
 const SelectedCardList: React.FC<Props> = (props: Props) => {
-  return <>
+  
+  return <div className="SelectedCardList">
+  <>
```

Comment:
이 <> 는 React의 구문 규칙 중 하나인 '각 컴포넌트는 단 하나의 부모 엘리먼트를 가지는 자식 엘리먼트들이 반환되어야 한다' 라는 규칙을 지키면서도 우아하게 이를 회피할 수 있는 [Fragments](https://reactjs-kr.firebaseapp.com/docs/fragments.html) 라는 기법입니다. 현재 PR 올려주신 걸 보면 'div' 가 Fragment 대신 최상위를 위치하고 있으니 이 태그를 지워도 괜찮을 것 같습니다.

Reply:
저기의 <>를 남겨둔 이유가 그 다음에 이어지는
```suggestion
 {
    props.uuids.map(x => <SelectedCardContainer onDeleteItem={props.onDeleteItem} uuid={x} />)
  }
```
를 쓰기 위해서는 비어있는 저 fragment를 부모로 두어야 하는 것 같더라구요. 로컬에서 해당 부분을 지우고 나니 컴파일이 되지 않는 것을 확인했습니다.

Reply:
--수정--
컴파일은 되는 것으로 확인되었습니다😂
그래도 저 fragment를 지우고 나니 FlatArray<Arr,?>를 비교하는 스택 깊이가 너무 깊다는 경고가 뜨는군요!